CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+# Code of Conduct
+
+Like the technical community as a whole, the various ESLint teams and communities are made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
```

Comment:
Box is currently paying me partly to work on ESLint, so I'm a professional according to your definition. :)